libs/feature/record/src/lib/data-view-permalink/data-view-permalink.component.ts

```diff
@@ -22,21 +22,21 @@ export const WEB_COMPONENT_EMBEDDER_URL = new InjectionToken<string>(
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
```

Comment:
I think it'd be good to define a `export type DATA_VIEW_TYPE = 'chart' | 'table' | 'map'` here and use it everywhere you use those values to prevent errors.

Reply:
I mentioned [here](https://github.com/geonetwork/geonetwork-ui/pull/921#pullrequestreview-2151320491) how I tested the web component in the docker container :-)

Reply:
I tried it, but the "share url" generated, which use localhost, make me go back automatically to the home page of the datahub...